dashboard/src/pages/ChapterEvents.vue

```diff
@@ -65,8 +65,7 @@ const past_events = createListResource({
   fields: ['*'],
   filters: [
     ['chapter', '=', route.params.id],
-    ['event_start_date', '<', new Date()],
```

Comment:
not sure why this was removed ?

Reply:
Cancelled events where start_data > today won't come here otherwise. In case even if they cancel an event by mistake or on purpose - would be good to keep them in log, as they can edit it if need after or repurpose it. 

I'd mentioned the case in description

Reply:
events with start_data > today coming in past_events dont make sense anyway, we can just have them in upcoming events section.

Reply:
I am not in favour of cancelled events coming in upcoming section, but will make the change for now